src/app/tracing/io/data-importer/xlsx-import/getWarnings.ts

```diff
@@ -45,7 +45,7 @@ function getTextForIssueGroup(issueGroup: IssueGroup): string {
                     : []),
             ]);
         default:
-            return "";
+            return issueGroup.msg;
```

Comment:
With the generic message I meant rather a:
`${issueGroup.msg} (sheet: ${issue.sheet}, col: ${issue.col}, row: ${issue.row})`
but this would result in a string array. (Col and row information should only be reported if available.)

In my eyes we should either leave the empty string and filter it afterwards or use rather the string array, because
Alex might use the dev version for presentation. If he imports and xls file and gets the warning 'Invalid value.' The user might be confused.

Reply:
I can see that.
What do you think would be a good placeholder for now, until we get more detailed requirements from the POs?

I could see Simply returning "Unknown Warning" or just filtering out the empty string as options, or of course the issueGroup.msg

None of these would be the permanent solution, but would be the fallback for any warnings we don't currently have a proper issue aggregator for.

Reply:
I would filter out the empty string.

Reply:
Done that, should we merge this @ruegenbfr?